Model/IO/Export/Nastran/NastranExporter/NastranExporter.h

```diff
@@ -0,0 +1,81 @@
+//
+// Created by Kiełbasa Karol on 28.03.2017.
+//
+
+#ifndef CPP_NASTRANEXPORTER_H
+#define CPP_NASTRANEXPORTER_H
+#include "../../MeshExporter.h"
```

Comment:
Are all the headers needed in this file? What is a rationale to include all this stuff here?